node/libs/concurrency/src/ctx/mod.rs

```diff
@@ -185,8 +185,8 @@ impl Ctx {
     ) -> CtxAware<impl 'a + Future<Output = OrCanceled<F::Output>>> {
         CtxAware(async {
             tokio::select! {
-                output = fut => Ok(output),
```

Comment:
what happened here?

Reply:
I introduced `fn Ok<T>(t: T) -> Result<T>` just so I can use `ctx::Ok(())` for convenience when needed. This messes with the existing references to `Ok` in the same module so I had to explicitly clarify them.